### PR TITLE
DAOS-13378 cart: debug multisend test

### PR DESCRIPTION
**Pull request by frostedcmos**
- Additional debug for multisend test
- Add HG_LOG_SUBSYS/HG_LOG_LEVEL to envariables parsed by cart utils

Skip-func-test-leap15: false

Required-githooks: true

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate watchers.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

File: src/tests/ftest/cart/util/cart_utils.py [hunk 1]
```diff
@@ -219,6 +219,14 @@ def get_env(self):
...
         ofi_domain = None
         ofi_share_addr = None
+        hg_log_subsys = None
+        hg_log_level = None
+
+        if "HG_LOG_SUBSYS" in os.environ:
+            hg_log_subsys = os.environ.get("HG_LOG_SUBSYS")
+
+        if "HG_LOG_LEVEL" in os.environ:
+            hg_log_level = os.environ.get("HG_LOG_LEVEL")
```
Comment (daltonbohning) [inline src/tests/ftest/cart/util/cart_utils.py:229]: FYI - the default is `None` so you could just do 
```suggestion
        hg_log_subsys = os.environ.get("HG_LOG_SUBSYS")
        hg_log_level = os.environ.get("HG_LOG_LEVEL")
```

**Review by daltonbohning — COMMENTED** [review]
Is this meant for review?

File: src/tests/ftest/cart/util/cart_utils.py (hunk 1 above)
Comment (frostedcmos) [reply src/tests/ftest/cart/util/cart_utils.py:229]: > FYI - the default is `None` so you could just do

Sorry, no was not meant for review, forgot to make it a draft, and was a quick copy/paste of existing envars how they were set.  It's good to know though, i ll make changes in a separate PR where I want to export those 2 additional envars in the test

Comment (frostedcmos) [reply src/tests/ftest/cart/util/cart_utils.py:229]: these changes moved to https://github.com/daos-stack/daos/pull/12161

**Pull request closed without merging**